cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,138 @@
+
+## Overview of PCI DSS
+
+When a system complies with PCI DSS, the system meets the goals of the standard by implementing a series of twelve requirements, as assessed by an independent PCI Qualified Security Assessor (QSA). The following table, which is published in Payment Card Industry Security Standards Council's [PCI DSS Quick Reference Guide, version 3.2.1](https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf), summarizes the goals and requirements of PCI DSS.
+
+<table>
+<tgroup cols="2">
+<thead>
+<tr>
+  <th>Goal</th>
+  <th>PCI DSS Requirement</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td>Build and Maintain a Secure Network and Systems</td>
+  <td><ol>
+          <li>Install and maintain a firewall configuration to protect cardholder data.</li>
+          <li>Do not use vendor-supplied defaults for system passwords and other security parameters.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Protect Cardholder Data</td>
+  <td><ol start="3">
+        <li>Protect stored cardholder data.</li>
+        <li>Encrypt transmission of cardholder data across open, public networks.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Maintain a Vulnerability Management Program</td>
+  <td><ol start="5">
+        <li>Protect all systems against malware and regularly update antivirus software or programs.</li>
+        <li>Develop and maintain secure systems and applications.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Implement Strong Access Control Measures</td>
+  <td><ol start="7">
+        <li>Restrict access to cardholder data by business need to know<./li>
+        <li>Identify and authenticate access to system components.</li>
+        <li>Restrict physical access to cardholder data.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Regularly Monitor and Test Networks</td>
+  <td><ol start="10">
+        <li>Track and monitor all access to network resources and cardholder data.</li>
+        <li>Regularly test security systems and processes.</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td>Maintain an Information Security Policy</td>
+  <td><ol start="12">
+        <li>Maintain a policy that addresses information security for all personnel.</li>
+  </td>
+</tr>
+</tbody>
+</tgroup>
+</table>
+
+{{ site.data.products.dedicated }} is certified by a PCI Qualified Security Assessor (QSA) to be compliant with PCI DSS [Self-Assessment Questionnaire D (SAQ-D)](https://listings.pcisecuritystandards.org/documents/SAQ_D_v3_Merchant.pdf), as long as you take the actions outlined in [How To Ensure Your Cluster's Compliance with PCI DSS](#how-to-ensure-your-clusters-compliance-with-pci-dss). SAQ-D has more stringent requirements than [SAQ-A](https://listings.pcisecuritystandards.org/documents/SAQ_A_v3.pdf), and many organizations require compliance with SAQ-D.
```

Comment:
@mdlinville @MichaelTrestman I think we should re-word to something like

to be compliant with [PCI DSS 3.2.1](https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v3-2-1-r1.pdf) 

IMO, we can drop the entire last sentence.